src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -73,12 +69,9 @@ export const SearchResults = ({
   });
 
   useEffect(() => {
```

Comment:
I know this was merged already, but this `useEffect` appears to no longer be necessary.